calico_node/startup/startup.go

```diff
@@ -539,6 +542,37 @@ func configureASNumber(node *api.Node) {
 	}
 }
 
+// getIPv6Pool return a random generated ULA IPv6 prefix generated following rfc4193#section-3.2.2
+// The Pool is generated with a concatenation of Unix timestamps + fe80:: base IPv6 hased with SHA-1
+func getIPv6Pool() string {
+	adm := os.Getenv("IP_AUTODETECTION_METHOD")
+	cidr := autoDetectCIDR(adm, 6)
+	eui := cidr.String()
+	if eui != "" {
+		date := fmt.Sprint(time.Now().Unix())
```

Comment:
I still think this is not the right approach for a normal server system where Felix runs, please use `crypto/rand` to get your random bytes.